src/main/java/org/sonarsource/scanner/maven/SonarQubeMojo.java

```diff
@@ -50,6 +50,7 @@
 @Mojo(name = "sonar", requiresDependencyResolution = ResolutionScope.TEST, aggregator = true)
 
 public class SonarQubeMojo extends AbstractMojo {
+    // TODO
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.scanner.maven%3Asonar-maven-plugin&branch=feature%2Fjanos%2Fjunk&issues=AWFQYhIYsS4rYBZqE3Gi&open=AWFQYhIYsS4rYBZqE3Gi)

[ISSUE_KEY]: AWFQYhIYsS4rYBZqE3Gi